src/main_app/db/services/settings_service.py

```
@@ -189,10 +189,17 @@ def create_setting(
-        setting = SettingRecord.query.filter_by(key=key).first()
-        if setting:
-            return delete_record_by_pk(SettingRecord, setting.id)
-        return False
+        record = SettingRecord.query.filter_by(key=key).first()
+        try:
+            if record:
+                db.session.delete(record)
+                db.session.commit()
+                return True
+            return False
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)